suite-native/module-send/src/components/AmountInputs.tsx

```diff
@@ -118,6 +118,7 @@ export const AmountInputs = ({ index, accountKey }: AmountInputProps) => {
                     inputRef={cryptoRef}
                     isDisabled={!isCryptoSelected}
                     networkSymbol={networkSymbol}
+                    onPress={!isCryptoSelected ? handleSwitchInputs : undefined}
```

Comment:
Otherwise it would be broken for testnet coins, I guess.
```suggestion
                    onPress={isFiatDisplayed && !isCryptoSelected ? handleSwitchInputs : undefined}
```

Reply:
Ok, it's not, because `!isFiatDisplayed => isCryptoSelected`